src/js/game/LevelMVC/LevelBlock.js

```diff
@@ -300,7 +300,7 @@ module.exports = class LevelBlock {
   }
 
   getIsEmptyOrEntity() {
-    return this.isEmpty || this.isEntity;
+    return this.isEmpty || this.isEntity || this.getIsLiquid();
```

Comment:
It feels like we're violating the nature of this check. Could we instead add a method like `getCanOcclude` that uses `getIsEmptyOrEntity` and `getIsLiquid`?

Reply:
Good call, I will fix!